coral-dragaction/src/scripts/DragAction.js

```diff
@@ -682,6 +687,45 @@ class DragAction {
     }
   }
 
+  /** @private */
+  _dragOnKey(event) {
+    switch (event.code) {
+      case 'Space':
+        this._dragEvents.dispatch('coral-dragaction:dragonkeyspace', {
+          detail: {
+            dragElement: this._dragElement
+          }
+        });
```

Comment:
Add the `focusout` event listener when the keyboard drag and drop operation starts.

```suggestion
        this._dragEvents.dispatch('coral-dragaction:dragonkeyspace', {
          detail: {
            dragElement: this._dragElement
          }
        });
        events.on(`focusout.DragAction${this._id}`, this._dragOnKeyEnd);
```